styleguide/_themes/derek/scss/components/_basement.scss

```diff
@@ -7,3 +7,11 @@
 .basement-copyright{
   color: $medium-gray;
 }
+.basement-region-list-americas {
```

Comment:
replace with mobile-first approach so we only have to put `column-count: 2` as an exception, instead of overriding it .